CONTRIBUTING.md

```diff
@@ -1,23 +1,20 @@
 # Contributing
 
-First read the overall project contributing guidelines. These are all
-included in the qiskit documentation:
+Qiskit is an open-source project committed to bringing quantum computing to
+people of all backgrounds. This page describes how you can join the Qiskit
+community in this goal.
 
-https://qiskit.org/documentation/contributing_to_qiskit.html
-
-## Contributing to Qiskit Terra
-
-In addition to the general guidelines there are specific details for
-contributing to terra, these are documented below.
 
 ### Contents
```

Comment:
The refactoring here seems to have broken the logical heading levels a little bit.  Not sure what heading structure you prefer, but at the moment we go from level 1 to level 3 then back to level 2, which isn't great for accessibility.

Reply:
ah good point, the heading levels where a bit janky to beign with so I tried to tidy them up a bit, but must have missed this one